glossary.md

```diff
@@ -14,50 +14,51 @@ Nếu một từ chưa có trong bảng thuật ngữ dưới đây, các bạn
 ## Anh - Việt
 
 | English                           | Tiếng Việt (Nếu bạn bổ sung thì KHÔNG căn chỉnh các dòng khác) | Thảo luận tại                |
-|-----------------------------------|----------------------------------------------------------------|------------------------------|
```

Comment:
Bạn lưu ý chỉ thay đổi dòng mà bạn thêm từ, không sửa toàn bộ các dòng khác, như thế rất khó review.